README.md

```diff
@@ -12,9 +12,15 @@ Sequelize is a promise-based Node.js/io.js ORM for Postgres, MySQL, MariaDB, SQL
 
 `npm install sequelize`
 
-Will install the latest version of sequelize (currently 2.0).
+From 3.0.0 and up Sequelize will follow SEMVER.
 
-[Upgrading to 2.0](https://github.com/sequelize/sequelize/wiki/Upgrading-to-2.0)
+## Backwards compatibility changes
```

Comment:
Even if people don't know what SEMVER is i hope they wouldn't expect major releases without changed. So the changelog is fine in my opinion.